src/handler/readiness.go

```diff
@@ -0,0 +1,49 @@
+package handler
+
+import (
+	"net/http"
+
+	"github.com/sirupsen/logrus"
+)
+
+// ReadinessHandler ReadinessProbe用function
+func ReadinessHandler(w http.ResponseWriter, r *http.Request) {
+	var rci RedisConnectionInterface
+	rci = new(RedisManager)
```

Comment:
この2行は一行にまとめられます（GitHubのSuggested changesがうまく機能するか不安）
```suggestion
var rci RedisConnectionInterface = new(RedisManager)
```